controllers/pagerdutyintegration/serviceorchestration_handler.go

```diff
@@ -96,14 +98,31 @@ func (r *PagerDutyIntegrationReconciler) handleServiceOrchestration(pdclient pd.
 		Namespace: pdi.Spec.ServiceOrchestration.RuleConfigConfigMapRef.Namespace,
 	}
 
-	orchestrationRuleConfigData, err := utils.LoadConfigMapData(r.Client,
-		serviceOrchestrationConfigMap, ServiceOrchestrationDataName)
-	if errors.IsNotFound(err) {
-		r.reqLogger.Info("service orchestration configmap rule not found, skip the following steps")
-		localmetrics.UpdateMetricPagerDutyServiceOrchestrationFailure(1, pdi.Name)
-		return nil
-	} else if err != nil {
-		return err
+	orchestrationRuleConfigData := ""
+
+	if utils.IsRedHatInfrastructure(cd) {
+		orchestrationRuleConfigData, err = utils.LoadConfigMapData(r.Client,
+			serviceOrchestrationConfigMap, RedHatInfraServiceOrchestrationDataName)
```

Comment:
I'd prefer this to be on one line, don't think it's easier to read this way

Reply:
I kept the formatting as it was before, it just moved into the if/else making it even longer. 
Personally I think the line split version is easier to read, but that's just personal preference.

Reply:
Sure, no problem, it's a matter of personal preference haha